src/components/Header/Header.test.tsx

```diff
@@ -0,0 +1,31 @@
+import Header from '@/components/Header/Header'
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+
+describe('Header', () => {
+  it('renders header', () => {
+    render(<Header />)
+
+    // Query header element
```

Comment:
Do we need these comments?

Reply:
That comment is for this 
```
// Query header element
const header = screen.getByRole('banner')
```
I can remove it if you think it's unecessary

Reply:
I think this is a good example of comments being noise.

To be it's obvious you are querying for the header by the code.

Reply:
alright, will remove this and other comments below :+1: